CONTRIBUTORS.md

```diff
@@ -1,10 +1,2 @@
-bourd0n | Nikita Aleksandrov | andone74@gmail.com  
```

Comment:
=___=
простите, меня не было на последних парах, я болела
и один "умник" мне сказал это сделать
сейчас все исправлю

5 апреля 2016 г., 22:56 пользователь bourd0n <notifications@github.com>
написал:

> In CONTRIBUTORS.md
> <https://github.com/bourd0nOrg/cats_world/pull/23#discussion_r58604665>:
>
> > @@ -1,10 +1,2 @@
> > -bourd0n | Nikita Aleksandrov | andone74@gmail.com
>
> @manesutopia <https://github.com/manesutopia> а зачем из файла
> CONTRIBUTORS.md удалила все строчки? Надо добавить только свою строчку в
> конец файла. Поправь пожалуйста.
> Предыдущий состав файла:
>
> bourd0n | Nikita Aleksandrov | andone74@gmail.com
> ONETUZ | Alexander Shvartsman | sashakaiser@yandex.ru
> Elfres | Anna Gruzdeva | anyagruzdeva@yandex.ru
> Dron74 | Kolyadin Andrey | avenger225@yahoo.com
> nadegda96 | Parfenova Nadegda| nadegda-sp@mail.ru
> Java2ME | Khadis Asfyandiyarov | Fisher5991@gmail.com
> Anastasiya4895 | Anastasiya Zinchenko | vesnushka4895@mail.ru
> S1ayerhs | Alexey Novikov | neimorr@yandex.ru
> Zigiz | Mezentsev Ivan | Zigiz1996@list.ru
> Zloto96 | Ignatov Ivan | Zloto1996@mail.ru
>
> Спасибо
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/bourd0nOrg/cats_world/pull/23/files/1c8b206b4bcc699f01ef5d57bd3dec26fe22ff57#r58604665>
>